src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/commands.py

```diff
@@ -0,0 +1,132 @@
+        g.command('create', 'create_or_update_authorization_rule',)
+        g.command('show', 'get_authorization_rule', exception_handler=empty_on_404)
+        g.command('list', 'list_authorization_rules', exception_handler=empty_on_404)
+        g.command('keys list', 'list_keys')
+        g.command('keys renew', 'regenerate_keys')
+        g.command('delete', 'delete_authorization_rule')
+        g.generic_update_command('update', getter_name='get_authorization_rule', setter_name='create_or_update_authorization_rule', custom_func_name='cli_namespaceautho_update')
+
+# Topic Region
+    with self.command_group('servicebus topic', sb_topic_util, client_factory=topics_mgmt_client_factory) as g:
+        g.custom_command('create', 'cli_sbtopic_create')
+        g.command('show', 'get', exception_handler=empty_on_404)
+        g.command('list', 'list_by_namespace', exception_handler=empty_on_404)
+        g.command('delete', 'delete')
+        g.generic_update_command('update', custom_func_name='cli_sbtopic_update')
+
+    with self.command_group('servicebus topic authorization-rule', sb_topic_util, client_factory=topics_mgmt_client_factory) as g:
+        g.command('create', 'create_or_update_authorization_rule')
+        g.command('show', 'get_authorization_rule', exception_handler=empty_on_404)
+        g.command('list', 'list_authorization_rules', exception_handler=empty_on_404)
+        g.command('keys list', 'list_keys')
+        g.command('keys renew', 'regenerate_keys')
+        g.command('delete', 'delete_authorization_rule')
+        g.generic_update_command('update', getter_name='get_authorization_rule', setter_name='create_or_update_authorization_rule', custom_func_name='cli_namespaceautho_update')
+
+# Subscription Region
+    with self.command_group('servicebus topic subscription', sb_subscriptions_util, client_factory=subscriptions_mgmt_client_factory) as g:
+        g.custom_command('create', 'cli_sbsubscription_create')
+        g.command('show', 'get', exception_handler=empty_on_404)
+        g.command('list', 'list_by_topic', exception_handler=empty_on_404)
+        g.command('delete', 'delete')
+        g.generic_update_command('update', custom_func_name='cli_sbsubscription_update')
+
+# Rules Region
+    with self.command_group('servicebus topic subscription rule', sb_rule_util, client_factory=rules_mgmt_client_factory) as g:
+        g.custom_command('create', 'cli_rules_create')
+        g.command('show', 'get', exception_handler=empty_on_404)
+        g.command('list', 'list_by_subscriptions', exception_handler=empty_on_404)
+        g.command('delete', 'delete')
+        g.generic_update_command('update', custom_func_name='cli_rules_update')
+
+# DisasterRecoveryConfigs Region
+    with self.command_group('servicebus georecovery-alias', sb_geodr_util, client_factory=disaster_recovery_mgmt_client_factory) as g:
+        g.command('create', 'create_or_update')
```

Comment:
If you intend for customers to use this command to update an existing alias, this command should be called `set`.

Reply:
implemented, changed create to set